utils/_features.py

```diff
@@ -1828,3 +1828,39 @@ def appsec_user_blocking(test_object):
         """
         pytest.mark.features(feature_id=254)(test_object)
         return test_object
+
+    @staticmethod
+    def kafkaspan_creationcontext_propagation_with_dd_trace_js(test_object):
+        """
+        [Kafka][Span Creation][Context Propagation] with dd-trace-js
+        https://feature-parity.us1.prod.dog/#/?feature=255
```

Comment:
Features did not exists yet, I've created them. This one got a different ID (255 was already taken)

```suggestion
        https://feature-parity.us1.prod.dog/#/?feature=259
```